Hyperbolic_surface_triangulation_2/doc/Hyperbolic_surface_triangulation_2/CGAL/Hyperbolic_fundamental_domain_2.h

```diff
@@ -0,0 +1,105 @@
+namespace CGAL{
+
+/*!
+\ingroup PkgHyperbolicSurfaceTriangulation2MainClasses
+
+Represents a fundamental domain of a closed orientable hyperbolic surface.
+The domain is given as a polygon \f$ P  \f$ represented by the list of its vertices in the Poincaré disk model,
+together with a pairing of the sides of \f$ P \f$.
+The \f$ n \f$-th side of \f$ P \f$ is the side between the \f$ n \f$-th and the \f$ (n+1) \f$-th vertex, where indices are modulo the number of vertices of \f$ P \f$.
+The side pairings are represented by a list of integers, such that if the \f$ n \f$-th integer of the list is \f$ m \f$, then the \f$ n \f$-th side is paired to the \f$ m \f$-th side.
+
+\tparam Traits is the traits class and must be a model of `HyperbolicSurfaceTraits_2` (default model: `Hyperbolic_surface_traits_2`).
+*/
+template<class Traits>
+class Hyperbolic_fundamental_domain_2 {
+public:
+  /// \name Types
+  /// @{
+  /*!
+  Point type.
+  */
+  typedef typename Traits::Hyperbolic_point_2                    Point;
+  /// @}
+  /// \name Creation
+  /// @{
+  /*!
+    Default constructor
+  */
+  Hyperbolic_fundamental_domain_2();
+
+  /*!
+    Constructor from vertices and pairings ranges.
+    @tparam PointRange a model of the concepts `RandomAccessContainer` whose `value_type` is `Point`.
+    @tparam PairingRange a model of the concepts `RandomAccessContainer` whose `value_type` is `int`.
+  */
+     template<class PointRange, class PairingRange>
+       Hyperbolic_fundamental_domain_2(PointRange & vertices, PairingRange & pairings);
+  /// @}
+
+  /// \name Access Functions
+  /// @{
+  /*!
+      returns the number of vertices (equivalently, the number of sides) of the domain.
+
+      \pre <code> is_valid() </code>
+  */
+  int size() const;
+
+  /*!
+      returns the \f$ i \f$-th vertex.
+
+      \pre <code> is_valid() </code>
+  */
+  const Point& vertex(int i) const;
+
+  /*!
+      returns the index of the side paired to the \f$ i \f$-th side.
```

Comment:
Just write i-th    that is no backtick no dollar sign.  Mael went through many CGAL files to unify that in [this PR](https://github.com/CGAL/cgal/pull/6849/files)